webapp/lib/controller.dart

```diff
@@ -350,7 +348,11 @@ void applyConfiguration(model.UserConfiguration config) {
 
   view.conversationPanelView.showCustomMessageBox(config.sendCustomMessagesEnabled ?? false);
 
-  view.conversationListPanelView.showConversationSelectCheckboxes(config.sendMultiMessageEnabled ?? false);
+  if (config.sendMultiMessageEnabled) {
```

Comment:
Would it make more sense to rename this config feature `mulitMessageActionEnabled` or something like that?
Or does it only enable sending multiple messages?

Reply:
yes, I think that would make sense - it also enables tagging multiple conversations and also marking multiple conversations as unread. I'll follow up with a PR to change this. thanks for the suggestion!